### ISSUE TITLE
"You are too heavy to mount horse" when wearing ornamental plate armor

### ISSUE THREAD

**Issue by l29ah** [CleverRaven/Cataclysm-DDA#31247]
# Steps To Reproduce

1. Wear ornamental plate armor and carry some minor stuff
2. Find a friendly horse
3. `r You are too heavy to mount horse`

# Expected behavior

https://en.wikipedia.org/wiki/Heavy_cavalry

# Versions and configuration

0.D-4039-g02eaddfe85-dirty
```
[
  "dda",
  "no_npc_food",
  "aftershock",
  "crt_expansion",
  "alt_map_key"
]
```

**Comment by davidpwbrown** [comment]
This probably means that the 20% of horses weight is a good maximum carrying capacity still..... but there needs to be horse variety.
Heavy Cavalry needs big strong horses.

**Comment by kevingranade** [comment]
This is working as intended, your average horse is not heavy calvary, and in fact I'd be sceptical of finding any heavy calvary capable horses. Feel free to provide any sources you find that say otherwise.

**Comment by I-am-Erk** [comment]
Huge pulling horses like Clydesdales aren't uncommon, they aren't warhorses but they should probably exist.

Someday of course we'll need the ability to give your horse some "horse alpha" mutagen...

**Comment by EirenexTheDragon** [comment]
Horse breeds are very different too in my mind. i dont think plate armor was a big thing in america so the horses were more suited for cowboys or so?

**Comment by l29ah** [comment]
https://www.horsetalk.co.nz/2015/08/18/too-fat-ride-horse/ suggests smaller horses can carry higher percentage of their weight, also it depends on whether there is/the saddle structure.
Also, the bigger was your horse, the heavier the armor it needed to carry, since its surface is bigger.

**Comment by dkargin** [comment]
According to wiki, the modern draught horses are even larger than it was necessary for medieval heavy cavalry needs.  Seems like another 300 years of active horse selection all over the plaet were not in vain and most horses we meet today can be considered 'alpha horse', compared to medieval ones.
There are some 'light' horses, but they exist for racing purposes. So the probability to meet such horses is even lower.

Long story short: regular draft horse is enough to carry both armored knight and his armored girlfriend.

**Comment by DracoGriffin** [comment]
Wrote this up on Discord and not seeing too much here:

<https://en.wikipedia.org/wiki/Heavy_cavalry#Renaissance_to_20th_century>
<https://en.wikipedia.org/wiki/Horses_in_warfare#Medium-weight> dispute on heavy weights as actually being medium weights
<https://en.wikipedia.org/wiki/Horses_in_the_Middle_Ages#Size_of_war_horses> more on heavy cavalry horses actually being smaller than believed.
<https://en.wikipedia.org/wiki/Destrier#Breeding_and_size> charger-type (heavy cavalry warhorses) were smaller than believed
<https://www.reddit.com/r/history/comments/685apv/do_any_ancient_war_horsedestrier_breeds_still/> more stuff suggesting size and current breeds.
<http://www.humanesociety.org/sites/default/files/archive/assets/pdfs/hsp/soaiv_07_ch10.pdf> a few years older, but gives details on the number of breeds in USA as well as some charts for New England and number of horses
<https://www.thesprucepets.com/most-popular-horse-breeds-1886146> Assuming the cost of purchase, care, training, stabling and so on for horses, it can be safely assumed that popularity equals ownership for the most part.

With that in mind, from that list, there are a few breeds hardy enough and the right size (or even larger), such as the <https://en.wikipedia.org/wiki/American_Quarter_Horse#Breed_characteristics>, <https://en.wikipedia.org/wiki/Thoroughbred#Breed_characteristics>, <https://en.wikipedia.org/wiki/Morgan_horse> (smaller range but official breed of Vermont), with the draft horses being the higher end.

In terms of weight limits, everything I've read suggest horses should only be carrying 20-25% of their body weight. For simpler terms, I'd suggest 20% for non-draft horses, and 25% for draft horses, whenever someone decides to granularize mon_horse into actual New England breeds (with maybe one rare breed).

**Comment by berangus** [comment]
A very quick Google search indicates 240 pounds as a recommended maximum total load for a typical horse. Bear in mind that this figure was probably devised with the horse's well-being in mind. That might not always be a priority for your survivor, but it's still a consideration. Maybe overburdening a horse should be possible, but with pain, stamina loss (I don't actually know if NPCs have stamina) or damage dealt to the horse over time?

Also there is the possibility of horse variants/breeds as others have pointed out. If we can have multiple dog breeds I suppose it's not that different for horses.

**Comment by mark7** [comment]
>  i dont think plate armor was a big thing in america

https://en.wikipedia.org/wiki/Conquistador shows images of conquistadors wearing plate armor as well as other, lighter, varieties.  Not sure how authoritative that is.

**Comment by rtizzy** [comment]
@kevingranade 

Sorry to necro but what is the current carrying capacity? 

To touch on the "20% of horses weight" some studies seem to indicate that this is the _safe_ ("comfortable for the horses") limit for how much a horse can carry. 

https://horseracingsense.com/how-much-weight-can-a-horse-carry/
https://extension.umn.edu/horse-care-and-management/guidelines-weight-carrying-capacity-horses#:~:text=The%20researchers%20found%20that%20an,Horse%20Management%20published%20in%201920.

The study referenced seems to be

https://www.sciencedirect.com/science/article/abs/pii/S0737080607004133

Note the title of "Light Riding Horses"

To quote from the introduction of this research:

>Comparably, a more recent study of 360 endurance race horses, primarily of Arabian breeding, evaluated horse and rider weight relationships. They analyzed weight loads as horse body weight independent of rider weight, rider weight independent of weight of the horse, and a rider weight ratio calculated as rider weight divided by the weight of the horse. They concluded that conditioned horses were able to carry 20 to 30% of their body weight for 100 miles.

This may deserve a separate issue (let me know if one exists). 

Thoughts: 

1. Horses should have traits and quality.
   1. Light, medium, heavy (weight of horse)
   2. Conditioned, not conditioned (or some varying quality)
   3. Health
   4. Distance travelled (Stamina?)
2. Perhaps some other qualities related to their genetics as mentioned in the study. 
  1. Some horses, regardless of weight, may be better genetically adapted to certain tasks like speed, endurance, overall health, etc. 
5. Should also have some concept of stamina similar to that of the players. 
   1. A player character could push a weak horse... but it might die. 
   2. A horse can be ridden "past it limits" like an engine. This would impact the horses health in some way
   3. Horses, like any animal, need rest. Rest the horse so it can heal up. 

This could be presented to the player in a lot of ways. 

It might even be worth considering a horse closer to the player character or a car (looking at code)

Imagine a real life human being: The heavier amount of weight you are carrying, the slower you can run (at maximum) and the less amount of time you can do it. 

There is some level where running (or even walking) is impossible. 

1. A human cannot carry 1 million pounds for any distance. 
2. A human can carry an extra 100 pounds for some distance but at a lower speed than they can carry 10 pounds. 

Sorry to ramble. I think this could be modeled better and have some more depth. It would be interesting to have horse as a valid alternatives (or supplement) to vehicle usage.

**Comment by EternalLaughter** [comment]
I believe that horses we have currently (550 KG large horse), should be able to carry 30% of their body weight.

Source 1: https://aerc.org/wp-content/uploads/2018/09/AERC_Rider_Handbook.pdf

"As a rule, the horse can carry up to 30% of his body weight, depending on his bone size, i.e., a 900 pound horse
should be able to carry approximately 250 pounds on his back"

Source 2: https://ker.com/equinews/horses-weight-carrying-ability-studied/

"This study was designed to test the weight-carrying ability of light-breed horses performing 45 minutes of light to moderate exercise, simulating a riding lesson for a student at an intermediate skill level. In this study, weight loads of 25 and 30% of the horse’s body weight were shown to influence work rate, heart rate, and lactate concentrations."

Source 3: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8151148/

"Serum creatine kinase activity was higher immediately post-exercise and 24 to 48 h post-exercise when animals carried a weight of 30% bodyweight ratio, but no change in enzyme activity was seen when horses carried a load of 15% and 20% bodyweight ratio "

Trying to parse the science of the stress factor, and weariness induced by that load in the science articles, is above my understanding, but from what i gather, 30% is tiresome for the horse, so you would need to rest your horse if you need to travel far and long for hours, which to be fair is very rare in CDDA.

The AERC (American Endurance Ride Conference) handbook i listed first, would be my goto source, as they sound more knowledgable in actual usage and training of horses.
They also do endurance runs of 50 and 100 miles. (With rest for the horses)

So my conclusion would be that most horses can use a 30% BW to carry capacity ratio.
